### PR TITLE
Add documentation for LCN cover platform

### PR DESCRIPTION
**Pull request by alengwenus**
**Description:**
Documentation is added for the LCN switch platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20288

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

File: source/_components/cover.lcn.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+date: 2018-11-01 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lcn.png
+ha_category: Cover
+ha_release: 0.89
+ha_iot_class: "Local Push"
+---
```
Comment (MartinHjelmare) [inline source/_components/cover.lcn.markdown]: We're merging platform pages that don't need details with the component page. We've already done this for the existing LCN pages.

We can remove the cover page.

**Review by MartinHjelmare — COMMENTED** [review]
Please rebase to solve conflicts. Add the config for the cover platform in the component. See below.

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/lcn.markdown [hunk 2]
```diff
@@ -10,13 +10,11 @@ footer: true
 logo: lcn.png
 ha_category:
   - Hub
+  - Cover
   - Light
   - Switch
 ha_release: 0.85
 ha_iot_class: "Local Push"
-redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/lcn.markdown]: This must be included

File: source/_components/lcn.markdown [hunk 3]
```diff
@@ -10,11 +10,13 @@ footer: true
 logo: lcn.png
 ha_category:
   - Hub
+  - Cover
   - Light
   - Switch
 ha_release: 0.85
 ha_iot_class: "Local Push"
 redirect_from:
+  - /components/cover.lcn/
```
Comment (MartinHjelmare) [inline source/_components/lcn.markdown]: We don't need a redirect from this page, since it never existed.

Comment (alengwenus) [reply source/_components/lcn.markdown]: Fixed. I got a little bit confused about how the docs should look like. Hope it‘s ok, now.

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @alengwenus! 🥇

**Pull request merged**